emily_landi/lib/greet.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+//ORIGINAL
+// module.exports = function(name){
+//   return 'hello ' + name;
+// };
+
+//BONUS
+var input = process.argv[2];
+console.log("input: " + input);
```

Comment:
console.logs checking values should normally be taken out before submitting code.